parts/dcosagentresourcesvmss.t

```diff
@@ -189,7 +189,7 @@
       "sku": {
         "capacity": "[variables('{{.Name}}Count')]",
         "name": "[variables('{{.Name}}VMSize')]",
-        "tier": "Standard"
+        "tier": "[variables('{{.Name}}VMSizeTier')]"
```

Comment:
Or is this so that everything still works even if the customer manually tries to edit and use a 'Basic_X' vm size?